pkg/controller/persistentvolume/persistentvolume_provisioner_controller.go

```diff
@@ -330,15 +330,19 @@ func (controller *PersistentVolumeProvisionerController) Stop() {
 	}
 }
 
-func newProvisioner(plugin volume.ProvisionableVolumePlugin, claim *api.PersistentVolumeClaim) (volume.Provisioner, error) {
+func newProvisioner(plugin volume.ProvisionableVolumePlugin, claim *api.PersistentVolumeClaim, pv *api.PersistentVolume) (volume.Provisioner, error) {
+	tags := make(map[string]string)
+	tags[cloudVolumeCreatedForClaimNamespaceTag] = claim.Namespace
+	tags[cloudVolumeCreatedForClaimNameTag] = claim.Name
+	if pv != nil {
```

Comment:
When would PV not be nil?  We first make the PV from the template and persist it with ClaimRef to the PVC.

It seems like nil PV here should be an error.